docs/asyncapi.yaml

```diff
@@ -0,0 +1,126 @@
+asyncapi: 3.0.0
+info:
+  title: API contract book for Single Front Door user communications
+  version: 1.0.0
+  description: Publish communications to users via the Single Front Door
+
+channels:
+  fcp-fd-comms:
+    address: fcp-fd-comms-request
+    description: |
+      Accepts requests for communication notifications via Azure service bus.
+    messages:
+      notifyMessage:
+        $ref: '#/components/messages/notifyMessage'
+
+operations:
+  SourceSystem/fcp-fd-comms.subscribe:
+    action: receive
+    channel:
+      $ref: '#/channels/fcp-fd-comms'
+    
+
+components:
+  messages:
+    notifyMessage:
+      title: gov.notify communications message
+      contentType: application/json
+      payload:
+        $ref: '#/components/schemas/govNotifyMessage'
+
+  schemas:
+    govNotifyMessage:
+      type: object
+      properties:
+        id:
+          type: string
+          description: Unique identifier for the event.
+          example: "abc123-uuid-123456"
+        source:
+          type: string
+          description: URI reference of the event's source.
+          example: "www.github.com/comms-requesting-service/app/routes/createEmail"
```

Comment:
I'd like this to be the service that sent the event, eg `ffc-ahwr-claim` so we can map it back